app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
Nice job so far😉. As a requirement, we should be able to confirm the registered user's email, and you seem to have tried to implement that😉. But the issue is, that the app breaks when I try to confirm these details😉. Please, do well to fix this issue and ensure everything works as required👍🏼.

<img width="895" alt="image" src="https://github.com/AlselwiAisha/BlogApp/assets/64643995/ea775935-9b41-4694-b0fd-c1e1b82cc9f7">